src/grid/rtl.scss

```diff
@@ -0,0 +1,43 @@
+
+// 响应列的水平位置偏移
+@mixin make-offset-rtl() {
+    @for $i from 1 through $grid-columns {
+        .#{$css-prefix}col-offset-#{$i}[dir="rtl"] {
+            margin-right: percentage($i / $grid-columns);
+            margin-left: 0;
```

Comment:
Do you think 0 is the correct default value to override `margin-left: percentage($i / $grid-columns);` of the class without `[dir="rtl"]` in main.scss?